keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -146,7 +146,7 @@ def __init__(
             ).as_backbone()
         )
 
-        self._prediction_decoder = prediction_decoder or cv_layers.NmsDecoder(
+        self._prediction_decoder = prediction_decoder or cv_layers.NonMaxSuppression(
```

Comment:
IMO it's become a bit confusing that we call this a "decoder" since it's not doing the relative-to-absolute coordinates transformation step.

Would it make sense to rename this concept in the OD models?

Reply:
Fair enough, again lets do that in a separate PR though.